deferred-call-manager/src/deploy.ts

```diff
@@ -8,8 +8,12 @@ import {
 } from '@massalabs/massa-web3';
 import { getScByteCode } from './utils';
 
-const rpcUrl = 'https://labnet.massa.net/api/v2:33035';
 const account = await Account.fromEnv();
+
+if (!process.env.JSON_RPC_URL_PUBLIC) {
+  throw new Error('JSON_RPC_URL_PUBLIC is not set');
+}
+const rpcUrl = process.env.JSON_RPC_URL_PUBLIC;
 const provider = Web3Provider.fromRPCUrl(rpcUrl, account);
```

Comment:
i think having a custom user defined rpc is not really mandatory here.
but if we want to support it maybe something like this would be ok:
```
    let provider;
    if (process.env.JSON_RPC_URL_PUBLIC) {
      provider = Web3Provider.fromRPCUrl(rpcUrl, account);
    } else {
      provider = Web3Provider.buildnet(account);
    }
    ```